UIKitDZ_App/UIKitDZ_App/PageViewController.swift

```diff
@@ -0,0 +1,225 @@
+//
+//  PageViewController.swift
+//  UIKitDZ_App
+//
+//  Created by coder on 13.10.2022.
+//
+
+import UIKit
+
+/// Экран с данными для PageControll
+final class PageViewController: UIPageViewController {
+    
+    // MARK: Visual Components
+    private enum  FirstPageElements {
+        static let firstLabel = "Track Your Cycle"
```

Comment:
```suggestion
        static let firstLabel = "Track Your Cycle"
```
это Label? поправь везде ниже